scripts/create_test

```diff
@@ -161,6 +161,8 @@ OR
                         help="Use this category key in the lookup in testlist.xml")
     parser.add_argument("--xml-testlist",
                         help="Use this testlist to lookup tests")
+    parser.add_argument("--testfile", nargs=1, default=None,
```

Comment:
This allows the user to specify multiple tests in a file rather than on the command line.